### PR TITLE
chore(codegen): use exclusion list for manual modules

### PR DESCRIPTION
**Pull request by diegomarquezp**
`generate-library-list.sh` now uses a list to indicate which handwritten autoconfigs to skip.
Contains changes from https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1498 (sanity check file creations).

### CODE REVIEW

**Review by emmileaf — COMMENTED** [review]
Thanks for putting together this more comprehensive exclusion list! Two quick questions on this PR's diff:
- Do you know why git picked up an empty edit for download-repos.sh?
- Could the sanity-check-related lines (from 1498) and monorepo-committish-related changes (from 1499) be dropped from this PR, since you have the other pull requests open for them?

**Comment by diegomarquezp** [comment]
@emmileaf The `download-repos.sh` comes from a `chmod 755` to mark the script as executable
I will remove the sanity check logic

**Review by meltsufin — APPROVED** [review]

File: spring-cloud-generator/generate-library-list.sh [hunk 1]
```diff
@@ -1,13 +1,7 @@
...
 
-#cmd line:: ./generate-library-list.sh -c v1.1.0
-
-while getopts c: flag
-do
-    case "${flag}" in
-        c) commitish=${OPTARG};;
-    esac
-done
+#cmd line:: ./generate-library-list.sh
```
Comment (emmileaf) [inline spring-cloud-generator/generate-library-list.sh:3]: Let's also remove this change since it's already made in #1499?

Comment (diegomarquezp) [reply spring-cloud-generator/generate-library-list.sh]: Change reverted - thanks!

**Pull request merged**